js/lib/api.js

```diff
@@ -28,15 +28,17 @@ window.PAPI = (function () {
         },
         decryptString: function (ciphertext, _key) {
             if(!ciphertext || !_key){
-                return '';
+                return false;
             }
             ciphertext = window.atob(ciphertext);
             var rp = {};
             try {
                 /** global: sjcl */
                 return sjcl.decrypt(_key, ciphertext, encryption_config, rp);
             } catch (e) {
-                throw e;
+                //throw e;
+                console.warn(e);
+                return false;
```

Comment:
Rework required.
You have to check all code that is using this decryptString function if you really want to change that behaviour.
There is code that requires on how the error handling with exceptions is currently handled. For example `decryptCredential` in line 44 of this file.